src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1632,6 +1632,14 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_Microsoft_PowerShell_Co
                 .EndTable());
         }
 
+        private const string GetPreReleaseStringScriptBlock = @"
```

Comment:
The `Get` prefix makes it sound like this is a function.  I think just calling it `PreReleaseStringScriptBlock` is better.

Reply:
done!

---
In reply to: [385355079](https://github.com/PowerShell/PowerShell/pull/11943#discussion_r385355079) [](ancestors = 385355079)